bootstrap.php

```diff
@@ -111,8 +111,8 @@ public function htmlUnclosed()
     if (isset($_GET['authenticated'])) {
       $dismissable_message = $this->dismissibleMessage('You successfully authenticated with Intercom');
     }
-    if (isset($_GET['enable_secure_mode'])) {
```

Comment:
This coomunicates with the wordpress_oauth server which needs to be updated too: https://github.com/intercom/wordpress-oauth/pull/11 @irvinebroque